packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -164,7 +167,7 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
               })
 
               wrappedErr.name = err.name
-              wrappedErr.stack = err.stack
+              wrappedErr.stack = $stackUtils.replacedStack(wrappedErr, err.stack)
```

Comment:
so the error is being reified, but we need the stack to be updated in the new wrapped error?

Reply:
Right, we want the message from`ran_origin_fn_reference_error`, but the stack of the original error.

We should probably update `$errUtils.throwErrByPath` to optionally accept a stack and name, so this manual massaging isn't necessary. Maybe in another PR.